File: problem_twenty_nine.js
// Perfect number !
var inp = 30
function perFect_Number(value){
    var count = 0
    for(var i = 1 ; i< value;i++){
        if (value % i == 0){
            count+=i
        }
    }
    if (count == value){
        console.log(value,"is perfect number!")
    }else{
        console.log(value,"is not perfect number!")
    }
}
perFect_Number(inp)

File: problem_twenty_nine.py
# perfect number !
value = int(input("Enter the number:"))
def Perfect_Number(value):
    count = 0
    for i in range(1,value,1):
        if value % i == 0:
            count+= i
    if count == value:
        print(value,"is perfect number !")
    else:
        print(value,"is not perfect number !")    
Perfect_Number(value)            